api/src/test/java/uk/gov/dhsc/htbhf/claimant/service/ClaimServiceTest.java

```diff
@@ -272,6 +272,21 @@ void shouldSendAdditionalPregnancyPaymentMessageWhenExistingDueDateUpdated() {
         shouldSendAdditionalPregnancyPaymentMessage(existingClaimant, newClaimant);
     }
 
+    @Test
+    void shouldReportClaimWhenTheClaimIsRejected() {
+        //given
+        Claimant claimant = aValidClaimant();
+        // an INELIGIBLE response will cause a claim to be rejected
+        EligibilityAndEntitlementDecision eligibility = aDecisionWithStatus(INELIGIBLE);
+        given(eligibilityAndEntitlementService.evaluateClaimant(any())).willReturn(eligibility);
+        ClaimRequest request = aClaimRequestForClaimant(claimant);
+
+        //when
+        ClaimResult result = claimService.createOrUpdateClaim(request);
+
+        verify(claimMessageSender).sendReportClaimMessage(result.getClaim(), eligibility.getDateOfBirthOfChildren(), ClaimAction.REJECTED);
```

Comment:
Very minor if you have `\\given` and `\\when` you should probably have `\\then` before this line.